### PR TITLE
fix: add retry to `google.auth.compute_engine._metadata.get()`

### PR DESCRIPTION
**Pull request by DaniilAnichin**
Initial fix of issue #211 was done in CL #323, but only for .ping()
This one is adding same behavior & tests for .get() method, as the problem still arises

Resolves: #211

### CODE REVIEW

**Comment by DaniilAnichin** [comment]
@dhermes can you take a look to finally make the #211 fixed?

**Comment by dhermes** [comment]
cc @busunkim96 

@DaniilAnichin I am no longer maintaining this library. Cheers

**Comment by lucasmbrown** [comment]
I can't tell who the maintainers currently are, but @theacodes or @busunkim96, would you be available to review this? It's caused us a few headaches so far (right now we're wrapping all our gets in our own retry logic). Thanks!

**Comment by busunkim96** [comment]
@lucasmbrown Thank you for the ping! I missed this PR. I'll take a look now.

File: setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 with io.open("README.rst", "r") as fh:
     long_description = fh.read()
 
-version = "1.10.0"
+version = "1.11.0"
```
[the remark this answers is not in the document]
Comment (DaniilAnichin) [reply setup.py]: Removed; was not sure about the flow

**Comment by DaniilAnichin** [comment]
Updated call count in tests; ping me if I need to rebase to current master

**Pull request merged**